api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -98,8 +98,13 @@ message DownstreamTlsContext {
   }];
 }
 
+// Custom config for customizing listener handshaker behavior.
+message CustomListenerHandshaker {
```

Comment:
remove this message? otherwise config will look weird like:

```
custom_listener_handshaker:
  typed_config:
    name: foo
    typed_config:
      "@type": fooType
      key: bar
```

Reply:
Done.